cves/CVE-2017-14170.yml

```diff
@@ -355,7 +400,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 834
 CWE_note:
```

Comment:
Do [CWE-20](https://cwe.mitre.org/data/definitions/20.html) and/or [CWE-770](https://cwe.mitre.org/data/definitions/770.html) also apply here?

Reply:
Doubly-so for CWE-20 from me. Classic lack of proper input handling (missing eof checks).

Reply:
I definitely think CWE-20 does apply here, but still think CWE-834 is the best one, as it actually describes the method by which the vulnerability is executed. I'll mention the others.